cmake/gen-libs/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+
+#
+# Copyright (C) The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Generate one static lib and one shared lib, copy them into
+# ${project_dir}/distribution for other indepdendent applications
+# to use.
+cmake_minimum_required(VERSION 3.4.1)
+
+set(CMAKE_VERBOSE_MAKEFILE on)
+
+set(lib_src_DIR ${CMAKE_SOURCE_DIR})
+
+set(lib_build_DIR $ENV{HOME}/tmp)
```

Comment:
I don't know how people would feel about dumping a random directory in their home...